core/src/options/core.rs

```diff
@@ -145,7 +150,9 @@ impl ParseData for Core {
         match self.data {
             Data::Struct(Fields {
                 style: Style::Unit, ..
-            }) => panic!("Core::parse_field should not be called on unit"),
+            }) => {
+                panic!("Core::parse_field should not be called on unit")
+            }
```

Comment:
Weird. Rustfmt decided to make this change when ran without #377, but llater when I added the empty `rustfmt.toml` it refuses to change it back.

Then if I manually change it back, it somehow has no complaints with it either? Weird 😅